src/components/molecules/AiInsightsTab/AiInsightsTab.tsx

```diff
@@ -0,0 +1,8 @@
+import {usePluginState} from '@plugins/pluginHooks';
+
+const AiInsightsTab = () => {
+  const [aiInsightsProps] = usePluginState<{id: string}>('executionDetailsTabs');
```

Comment:
Does the `AiInsightsTab` needs to know the current `id` of execution details tabs? It is enough to know that it's mounted (and it's possible to handle without state)

Reply:
it will need to know the `executionId`, because we will use it in the API request, to fetch AI insights specifically for this execution

Reply:
Ahh, right, `id` mislead me 🙂 Probably worth to call it `executionId` then (and we could expose more data, including entity details)